ui/infra-compare/InfraCompareTableRow.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class InfraCompareTableRow extends React.PureComponent {
+  getColorClass = (data, type) => {
+    const { className, isRegression, isImprovement } = data;
+    if (type === 'bar' && !isRegression && !isImprovement) return 'secondary';
+    if (type === 'background' && className === 'warning')
+      return `bg-${className}`;
+    if (type === 'text' && className) return `text-${className}`;
+    return className;
+  };
+
+  render() {
+    const { rowLevelResults } = this.props;
```

Comment:
The code below would be a bit cleaner if you deconstructed this value more.  You could do:

    const { rowLevelResults: { originalValue, newValue, newDataPoints } } = this.props;

Then you don't have to use the dot notation all over the place in the rest of the code.